python/tutorials/01-vector-add.py

```diff
@@ -42,14 +42,13 @@ def add_kernel(x_ptr,  # *Pointer* to first input vector.
     block_start = pid * BLOCK_SIZE
     offsets = block_start + tl.arange(0, BLOCK_SIZE)
     # Create a mask to guard memory operations against out-of-bounds accesses.
-    mask = offsets < n_elements
     # Load x and y from DRAM, masking out any extra elements in case the input is not a
     # multiple of the block size.
-    x = tl.load(x_ptr + offsets, mask=mask)
-    y = tl.load(y_ptr + offsets, mask=mask)
+    x = tl.load(x_ptr + offsets, cache_modifier='.cg')
```

Comment:
tutorial is meant to be simple and not necessarily optimal, I think this belongs more to an optimized kernel

Reply:
Ah, sorry this is not meant to land as-is. Just showcasing the optimization needed for better bandwidth.